Jenkinsfile

```diff
@@ -96,6 +96,32 @@ pipeline {
                         runScm()
                     }
                 }
+
+                stage('build-jvm-native-libs') {
```

Comment:
As already discussed we should maybe guard this with a flag, so it is easy to selectively build this only for publications and if you would like to verify it on a PR build.